base/Dockerfile.ubi8

```diff
@@ -79,6 +79,7 @@ ARG PYTHON_CONFLUENT_DOCKER_UTILS_INSTALL_SPEC="git+https://github.com/confluent
 RUN microdnf --nodocs install yum \
     && rpm --import https://www.azul.com/files/0xB1998361219BD9C9.txt \
     && yum --nodocs -y install https://cdn.azul.com/zulu/bin/zulu-repo-1.0.0-1.noarch.rpm \
+    && yup -y upgrade \
```

Comment:
hey @janjwerner-confluent ,

Do you mean "yum"? ;)
Also, should we use `yum update` instead of `yum upgrade`? 🤔 

`yum update` updates all packages to their latest version within the same major release, while `yum upgrade` upgrades all packages to the latest available version, including packages that require a major version change, and may result in significant changes to the system. It's recommended to use `yum update` for routine updates and yum upgrade with caution, as it can potentially cause compatibility issues or unexpected changes to the system.

Reply:
ahh fat fingers, of course yum, the difference I was looking for was cleaning up old packages.